config/index.js

```diff
@@ -189,42 +189,70 @@ module.exports = {
   LOCATIONS_BATCH_SIZE: process.env.LOCATIONS_BATCH_SIZE || 40,
   E2E: {
     BASE_URL: process.env.E2E_BASE_URL || BASE_URL,
-    ROLES: {
-      POLICE: {
-        username: process.env.E2E_POLICE_USERNAME,
-        password: process.env.E2E_POLICE_PASSWORD,
-      },
+    USERS: {
       COURT: {
-        username: process.env.E2E_COURT_USERNAME,
-        password: process.env.E2E_COURT_PASSWORD,
+        locations: ['ABDRCT', 'BDFRCT'],
+        name: 'End-to-end Court',
+        authorities: ['ROLE_PECS_COURT'],
+        username: 'E2E_COURT',
+        password: 'ZZDBm6Bf',
```

Comment:
If we are steering into have completely different local passwords to the ones in environments, can we make them memorable? This way when we're running locally we won't have to scroll to line `194`-`255` of a `config` file to figure out the correct password.

Reply:
Yep, not against that. Could just use a simple format seeing as it's only for mock test purposes anyway.